addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -409,6 +409,7 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
             this.dataForEmailTo = dataForValues['email_to'];
         }
         this.defaultEmailToValue = "info@yourcompany.example.com";
+        this.$target[0].querySelector(".s_website_form_send").classList.add("oe_unremovable")
```

Comment:
Can't we add class ```oe_unremovable``` in xml [here](https://github.com/odoo/odoo/blob/15.0/addons/website/views/snippets/s_website_form.xml#L11) and [here](https://github.com/odoo/odoo/blob/15.0/addons/website/data/website_data.xml#L132)